action.yml

```diff
@@ -26,6 +26,13 @@ inputs:
   release-branches:
     description: Configures the branches that Semantic Release will release to.
     required: false
+outputs:
+  version:
+    description: Released version.
```

Comment:
Can we document the format by providing an example? I don't know if e.g. it has `v` in front of it or not (`v1.0.0` vs `1.0.0`). If it has `v`, I think we should remove it. Why? Because it's easier to add it if someone wants it, rather than to remove it.

Reply:
it produces values without `v` : `1.10.1`